### PR TITLE
Add streetAction as variable of onChange

### PR DESCRIPTION
**Pull request by ryotogashi**

### CODE REVIEW

**Comment by ryotogashi** [comment]
@axross Can you check when you have time?

File: components/StreetActionSelector/StreetActionSelector.tsx [hunk 1]
```diff
@@ -87,24 +102,32 @@ export default function StreetActionSelector({
...
-  fold = "f",
-  checkOrCall = "c",
-  betOrRaise = "r",
+export enum StreetAction {
+  fold = "FOLD",
+  check = "CHECK",
+  call = "CALL",
+  bet = "BET",
+  raise = "RAISE",
 }
```
Comment (axross) [inline components/StreetActionSelector/StreetActionSelector.tsx:131]: Could you replace them with GameStreetAction's ones?

File: components/StreetActionSelector/StreetActionSelector.tsx [hunk 2]
```diff
@@ -34,24 +34,37 @@ export default function StreetActionSelector({
...
   const onCheckOrCallButtonClick = React.useCallback(() => {
-    setAction(StreetAction.checkOrCall);
-    onChange(tableMaxBetSize);
+    if (tableMaxBetSize === 0) {
+      setAction(StreetAction.check);
+      onChange(StreetAction.check, tableMaxBetSize);
+    } else {
+      setAction(StreetAction.call);
+      onChange(StreetAction.call, tableMaxBetSize);
+    }
```
Comment (axross) [inline components/StreetActionSelector/StreetActionSelector.tsx:47]:
```suggestion
    const action = tableMaxBetSize === 0 ? StreetAction.check : StreetAction.call;

    setAction(action);
    onChange(action, tableMaxBetSize);
```

**Comment by ryotogashi** [comment]
@axross Thank you for your review. I fixed them. Can you check them again?

File: components/StreetActionSelector/StreetActionSelector.test.tsx [hunk 3]
```diff
@@ -50,7 +51,7 @@ describe("<StreetActionSelector>", () => {
     ).toThrow();
   });
 
-  it('calls onChange() with props.previousBetSize when "Fold" button is clicked', async () => {
+  it('calls onChange() with StreetAction.fold and props.previousBetSize when "Fold" button is clicked', async () => {
```
Comment (axross) [inline components/StreetActionSelector/StreetActionSelector.test.tsx:54]:
```suggestion
  it('calls onChange() with GameStreetActionType.fold and props.previousBetSize when "Fold" button is clicked', async () => {
```

**Pull request merged**